addons/html_editor/static/src/fields/html_field.js

```diff
@@ -27,9 +31,14 @@ function computeContainsComplexHTML(value) {
 
 export class HtmlField extends Component {
     static template = "html_editor.HtmlField";
+    static defaultProps = {
+        dynamicPlaceholder: false,
+    };
```

Comment:
we always define defaultProps after props (it s juste a convention)

Reply:
sure